test/controllers/CancelDeclarationControllerSpec.scala

```diff
@@ -37,7 +37,6 @@ class CancelDeclarationControllerSpec extends CustomExportsBaseSpec {
   "CancelDeclarationController" should {
     "return 200 with a success" in {
       authorizedUser()
```

Comment:
There is authorizedUser() invocation in every test, so I'd suggest to use `BeforeAndAfter` and `before {...}` clause.